web/src/features/AppConfig/components/AppConfig.tsx

```diff
@@ -752,66 +752,67 @@ class AppConfig extends Component<Props, State> {
     return (
       <div className="flex flex-column u-paddingLeft--20 u-paddingBottom--20 u-paddingRight--20 alignItems--center">
         <KotsPageTitle pageName="Config" showAppSlug />
-        {fromLicenseFlow && app && (
-          <Span size="18" weight="bold" mt="30" ml="38">
-            Configure {app.name}
-          </Span>
-        )}
-        <div className="flex" style={{ gap: "20px" }}>
-          <div
-            id="configSidebarWrapper"
-            className="config-sidebar-wrapper card-bg clickable"
-          >
-            {configGroups?.map((group, i) => {
-              if (
-                group.title === "" ||
-                group.title.length === 0 ||
-                group.hidden ||
-                group.when === "false"
-              ) {
-                return;
-              }
-              return (
-                <div
-                  key={`${i}-${group.name}-${group.title}`}
-                  className={`side-nav-group ${
-                    this.state.activeGroups.includes(group.name) ||
-                    group.hasError
-                      ? "group-open"
-                      : ""
-                  }`}
-                  id={`config-group-nav-${group.name}`}
-                >
+        <div className="tw-flex tw-flex-col tw-mx-48">
+          {fromLicenseFlow && app && (
```

Comment:
added some div - flex to fix the alignment of the app name and the config section.